larray/core.py

```diff
@@ -3905,7 +3905,7 @@ def __init__(self,
                 raise ValueError("length of axes %s does not match "
                                  "data shape %s" % (axes.shape, data.shape))
 
-        # Because __getattr__ and __setattr__ have been rewritten
+        # Because __getattr__ and __setattr__ have been overridden
```

Comment:
even though it is indeed the case, `__setattr__` should actually not be overridden (ever since we changed `__getattr__` to return axes). It might be better to fix that in a separate PR though, because most (all?) `object.__setattr__` could/should be replaced.